documentation/Functions.md

```diff
@@ -244,11 +249,18 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
 ***
 
 ### createFolder <a name="createFolder"></a>
+This function is used when you want to use an other command in a location which does not exist at the moment 
```

Comment:
I would simply write: This function creates a new folder
And then you can mention that one can use this when wanting to run other commands (e.g. createFile) in locations that does not exist at the moment